tests/network-ovn

```diff
@@ -2127,6 +2127,57 @@ ovn_leases_tests() {
         lxc network delete lxdbr0
 }
 
+ovn_vlan_uplink_tests() {
+    echo "==> vlan uplink"
+
+    sudo ovs-vsctl set open_vswitch . \
+        external_ids:ovn-remote=unix:/var/run/ovn/ovnsb_db.sock \
+        external_ids:ovn-encap-type=geneve \
+        external_ids:ovn-encap-ip=127.0.0.1
+
+    lxc network create lxdbr0 \
+        ipv4.address=10.10.10.1/24 ipv4.nat=true \
+        ipv4.dhcp.ranges=10.10.10.2-10.10.10.199 \
+        ipv6.address=fd42:4242:4242:1010::1/64 ipv6.nat=true \
+        ipv6.ovn.ranges=fd42:4242:4242:1010::200-fd42:4242:4242:1010::254
```

Comment:
why using ipv6.ovn.ranges on the lxdbr0 network?

Reply:
Removed